src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -130,7 +152,16 @@ class MsalAuthorizeService implements AuthorizeService {
                 if (this._settings.defaultAccessTokenScopes?.length > 0) {
                     // This provisions the token as part of the sign-in flow eagerly so that is already in the cache
                     // when the app asks for it.
-                    await this._msalApplication.acquireTokenSilent(request);
+                    const account = this.getAccount();
+                    if (!account) {
+                        return this.error("No account to get tokens with.");
```

Comment:
```csharp
                        return this.error("No account to get tokens for.");
```
I'm not sure if my english is broken, but with doesn't sound correct there. Feel free to ignore if you think it's not right

Reply:
Your recommendation makes sense.